README.md

```diff
@@ -1,3 +1,6 @@
 # Northcoders News API
 
 For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+
+1. Before running any filesthat require access to the database, user will need to connect to database by setting PGDATABASE = database_name_here, either in terminal or in a script.
+
```

Comment:
These instructions could include more details on creating the .env files that provide this info